src/main/java/org/example/NumberUtils.java

```diff
@@ -0,0 +1,26 @@
+package org.example;
+
+import org.jetbrains.annotations.NotNull;
+
+import java.util.*;
+
+public class NumberUtils {
+
+    public int getSum(@NotNull List<Integer> numbers){
```

Comment:
Using @NotNull would bring us some safety and also make code a bit harder to read :)